src/components/SwipeThumb/index.js

```diff
@@ -189,14 +189,15 @@ const SwipeThumb = props => {
     railStyles,
     screenReaderEnabled,
     title,
+    isRTL
   } = props;
 
   const panStyle = {
     backgroundColor,
     borderColor,
     width: animatedWidth,
     ...railStyles,
-    ...(enableRightToLeftSwipe ? styles.containerRTL : styles.container),
+    ...(enableRightToLeftSwipe ? ( isRTL ? styles.container: styles.containerRTL ) : (isRTL ? styles.containerRTL : styles.container)),
```

Comment:
Can you please try below things
1. Delete new prop 'isRTL' and try to use `I18nManager.isRTL`
2. Rename existing prop `enableRightToLeftSwipe` to `enableReverseSwipe`
3. By default for all RTL layouts swipe button should work as right to left swipe
Without any code changes, if we switch LTR to RTL then demo app should look like below screenshot
<img width="300" src="https://user-images.githubusercontent.com/7910545/92436997-931a7200-f16b-11ea-8649-c3cd5b7171a4.png" />